test/e2e/tests/edit-gas-fee.spec.js

```diff
@@ -216,7 +216,10 @@ describe('Editing Confirm Transaction', function () {
 
         // open dapp and connect
         await connectDappWithExtensionPopup(driver);
-        await driver.clickElement({ text: 'Send', tag: 'button' });
+        await driver.clickElement({
+          text: 'Send EIP 1559 Transaction',
```

Comment:
Do we want to also still do a test for `Send Legacy Transaction`?

Reply:
Yes, that test lives in send-eth.spec.js

Reply:
Ah ok!